go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/freiheit-com/kuberpult
 
-go 1.19
+go 1.21
```

Comment:
Why not directly go to 1.22?

Reply:
that was actually an accident.
I'm fine with upgrading to 1.22, but I would do this in a separate PR.